docs/guide/internationalization.md

```diff
@@ -268,6 +268,34 @@ You can also assign a Default content availability flag to Content Types (availa
 
 Note that if a language is not provided in the list of prioritized languages and it is not the Content item's first language, the URL alias for this content in this language will not be generated.
 
+### SiteAccess-aware Repository - optional
```

Comment:
This can probably also go into above mentioned page, specifically part of, or next to:
https://doc.ezplatform.com/en/2.2/api/public_php_api_browsing/#loading-content-in-different-languages

For instance this bullet point of that section is referring to the SiteAccessAware repository change we do know:
`- Note: As of v2.0 this is planned to be done for you when you don't specify languages.`

So that note can be changes to referring to: as of 2.2, ContentView and Parameters resolvers use this by default, as of 2.3 auto-wiring of services will most likely have this by default also it seems:  https://github.com/ezsystems/ezpublish-kernel/pull/2380

And then the note can say read more below in SiteAccess Aware section, which for the most part can be what you propose here in internationalization.md.

But while on this block in Browsing, finding, viewing, the example already there:
`$content->getFieldValue( $fieldDefinition->identifier, 'fre-FR' )`

becomes the following with SiteAccess aware repo:
`$content->getFieldValue( $fieldDefinition->identifier )`

--

**For the text proposed here, maybe something like:**


The SiteAccess-aware repository is an instance of the eZ Platform Repository API which injects prioritised languages for you when you load data *(Content, Location, Content type, etc.)*, and don't specify languages to load yourself in API arguments.

Currently it is:
- Available as a private service `ezpublish.siteaccessaware.repository`, including for the other Repository services.
- Used out of the box on parameter converters for Content & Location
- Used out of the box on ContentView
- Planned to be used for autowired API services as of 2.3, and as of 3.0 will be returned on `ezpublish.api.*` service id's

Example of code before using SiteAccess aware repository:
```php
$content = $this->contentService->loadContent(
    42,
    $this->configResolver->getParameter('languages')
);

$name = $content->getVersionInfo()->getName();
$value = $content->getFieldValue('body')
```

Becomes:
```php
$content = $this->contentService->loadContent(42);

$name = $content->getVersionInfo()->getName();
$value = $content->getFieldValue('body')
```